todo_app/data/trello_items.py

```diff
@@ -0,0 +1,120 @@
+import requests, json, os
+
+trello_key = os.environ.get('TRELLO_KEY')
+trello_token = os.environ.get('TRELLO_TOKEN')
+trello_board = os.environ.get('TRELLO_BOARD')
+
+
+def get_lists():
+    """
+    Fetches all the lists for the board from trello.
+
+    Returns:
+        dictionary: key value pairs of board id and board name, as well as board name and id (for reverse lookup)
+    """    
+    lists = {}
+    url = 'https://www.trello.com/1/boards/{0}/lists?key={1}&token={2}'.format(trello_board, trello_key, trello_token)
```

Comment:
This works, but their docs actually reference `api.trello.com`, not `www.trello.com`.

Interestingly, the www host seems to redirect to just `trello.com`... I'd just stick to api.trello.com. Seems most likely to keep working.